datalad/interface/common_opts.py

```diff
@@ -269,6 +269,11 @@
     See https://git-annex.branchable.com/git-annex-groupwanted/ for more information""",
     constraints=EnsureStr() | EnsureNone())
 
+annex_ignore_opt = Parameter(
+    args=("--annex-ignore",),
+    metavar='BOOL',
```

Comment:
It is generally not beneficial to declare `metavar` AND `constraints`. Here is the help resulting from this implementation:

```
Usage: datalad siblings [--annex-ignore BOOL]
```

when `metavar` is removed:

```
Usage: datalad siblings [--annex-ignore {true|false}]
```

In many cases the constraints are capable of explaining themselves.